starknet_py/common.py

```diff
@@ -30,6 +30,15 @@ def create_compiled_contract(
     return definition
 
 
+def _create_compiled_contract(compiled_contract: str) -> CompiledContract:
+    """
+    Creates CompiledContract from already compiled contract.
+
+    :return: a CompiledContract instance.
+    """
+    return cast(CompiledContract, CompiledContractSchema().loads(compiled_contract))
+
+
 def create_contract_class(
```

Comment:
Perhaps this could get deprecated? Are there valid use cases for the user to create `ContractClass`?

Reply:
Why not leave it? It's used in some places around the library.

Reply:
I think 99% of these use cases are create by the classmethod now.